IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml

```diff
@@ -4875,31 +4878,14 @@ components:
           $ref: '#/components/schemas/LegacyReservationParameterDef'
         coveredSection:
           $ref: '#/components/schemas/TravelSectionDef'
-        requiredPersonalData:
-          $ref: '#/components/schemas/PersonalDataConstraintDef'          
       required:
         - id
         - type
         - regionalConstraint
         - combinationConstraint
         - travelValidityConstraint
         - afterSalesCondition
-      additionalProperties: false
-      
-    RegulatoryConditionsDef:
```

Comment:
this is needed, maybe you used an old version as baseline

Reply:
Adressed

Reply:
indeed. I will manually put it back in my next pull request